srcs/HTTPRequest.cpp

```diff
@@ -27,7 +27,7 @@ HTTPRequest::HTTPRequest(std::string raw_request)
 	_uri = raw_request.substr(0, raw_request.find(" "));
 	raw_request = raw_request.substr(raw_request.find_first_of(" \t")+1);
 	extractVersion(raw_request);
-	raw_request = raw_request.substr(raw_request.find_first_of("\n")+1);
+	raw_request = raw_request.substr(raw_request.find_first_of("\r\n")+1);
```

Comment:
Pour tout le parsing de la requête, je pense que c'est trop sujet aux erreurs de partir sur cette méthode pour parser.
La par exemple, si la request line commence par un espaces, ça crash.
Vu que le protocole est très strict sur la grammaire d'une requête, il faut tirer avantage de ça.

RFC 2617 : 5.1.1
The Request-Line begins with a method token, followed by the
Request-URI and the protocol version, and ending with CRLF. The
elements are separated by SP characters. No CR or LF is allowed
except in the final CRLF sequence.
    Request-Line   = Method SP Request-URI SP HTTP-Version CRLF
    
Donc tu peux faire quelque chose de plus strict en raccord avec ça.